docs/gh_pages/docs/retrieval_chain.md

```diff
@@ -0,0 +1,200 @@
+# Identity-enabled RAG using PebbloRetrievalQA
+
+PebbloRetrievalQA is a Retrieval chain with Identity & Semantic Enforcement for question-answering against a vector database.
+
+This document covers how to retrieve documents with Identity & Semantic Enforcement.
+
+To start, we will load documents with authorization metadata into an in-memory Qdrant vector database we want to use and then use it as a retriever in
```

Comment:
Can you adjust this and the rest of the document to be written in the third person?  Instead of "we will load" .. switch to phrases like "PebbloRetrievalQA chain uses documents loaded with authorization metadata using one of the many support Vector databases"